app/controllers/api/v1/users/sessions_controller.rb

```diff
@@ -1,10 +1,14 @@
 module Api
   module V1
     module Users
-      class SessionsController < Devise::SessionsController
+      class SessionsController < DeviseTokenAuth::SessionsController
         respond_to :json
 
-        private
+        protected
+
+        def render_create_success
+          render json: { message: 'Logged in successfully.' }, status: :created
```

Comment:
Since this is a sign in, WDYT of returning the user information here to avoid the client having to do a second request for getting the user info?

Reply:
Also, I think we should not return created (201) as the status here, but 200